jena-geosparql/src/test/java/org/apache/jena/geosparql/implementation/jts/CustomCoordinateSequenceTest.java

```diff
@@ -0,0 +1,85 @@
+package org.apache.jena.geosparql.implementation.jts;
```

Comment:
No license header. Please could you add one - the text from one of the adjacent files.

(It is better if the contributor adds the license rather than someone else add it. PS The build checks!)